build/docker/Dockerfile

```diff
@@ -1,10 +1,19 @@
-FROM golang:alpine
+FROM golang:1.11-alpine AS build
 
-WORKDIR /go/src/github.com/honeyscience/honeydipper
+WORKDIR /go/src/github.com/honeydipper/honeydipper
 
 RUN apk add git gcc libc-dev
 COPY . .
 RUN go get -u github.com/golang/dep/cmd/dep && dep ensure
 RUN go install ./...
 
+FROM alpine
+
+LABEL description="Honeydipper - an event-driven orchestration framework" \
+      org.label-schema.vcs-url=https://github.com/honeydipper/honeydipper \
+      org.label-schema.schema-version="1.0"
+
+WORKDIR /app
+COPY --from=build /go/src/github.com/honeydipper/honeydipper ./
```

Comment:
this copy will only copy the the source code, but not the binaries.  The binaries are probably in `$GOPATH/bin` some where, so we need to copy those for this image to work.  I might be wrong, but we should try this on local machine first.

Reply:
You’re right

Reply:
Should be correct now. See output in updated PR comment